src/i18n/en.json

```diff
@@ -12,19 +12,19 @@
   },
   "icrc2_approve": {
     "title": "Authorize another address to withdraw from your account",
-    "address_is_allowed": "The following address is allowed to withdraw from your account:",
+    "address_is_allowed": "The following address is allowed to withdraw from your account",
     "your_subaccount": "Your subaccount",
     "your_account": "Your account",
     "requested_withdrawal_allowance": "Requested withdrawal allowance",
     "withdrawal_allowance": {
       "some": "Current withdrawal allowance",
-      "none": "The allowance will be set to {} {} independently of any previous allowance. Until this transaction has been executed the spender can still exercise the previous allowance (if any) to it's full amount."
+      "none": "The allowance will be set to {0} {1} independently of any previous allowance. Until this transaction has been executed the spender can still exercise the previous allowance (if any) to it's full amount."
```

Comment:
Just for explicity, shall it be `amount` and `symbol`?

Reply:
Good idea! Done.